src/main/resources/scripts/-examples/events.sk

```diff
@@ -0,0 +1,44 @@
+
+#
+# This example listens for players joining and leaving.
+# Alters the default message when they do.
+#
+
+on join:
+    set the join message to "Oh look, %player% joined! :)"
+
+on quit:
+    set the quit message to "Oh no, %player% left! :("
+
+#
+# This example cancels damage for players if they have a specific permission.
+# If they don't, tell them how much damage they took.
+#
+
+on damage:
+    victim is a player
+    if the victim has permission "skript.example.damage":
+        cancel the event # Stops the default behaviour - the victim taking damage.
+    else:
+        send "Ouch! You took %damage% damage." to the victim
+        add damage to {damage::%uuid of victim%::taken}
```

Comment:
Is there a purpose for these variables? Maybe adding on some sort of leaderboard command could be nice.

Reply:
Originally I was thinking of something like that, but didn't know whether it was within the scope of something like this.

Reply:
I think it could go either way whether it should be added